### PR TITLE
KAFKA-3043: Replace request.required.acks with acks in docs.

### PR DESCRIPTION
**Pull request by sasakitoa**
In Kafka 0.9, request.required.acks=-1 which configration of producer is replaced by acks=all, 
but this old config is remained in docs.

### CODE REVIEW

File: docs/configuration.html [hunk 1]
```diff
@@ -106,8 +106,8 @@
       <td>min.insync.replicas</td>
       <td>1</td>
       <td>min.insync.replicas</td>
-      <td>When a producer sets request.required.acks to -1, min.insync.replicas specifies the minimum number of replicas that must acknowledge a write for the write to be considered successful. If this minimum cannot be met, then the producer will raise an exception (either NotEnoughReplicas or NotEnoughReplicasAfterAppend).
-      When used together, min.insync.replicas and request.required.acks allow you to enforce greater durability guarantees. A typical scenario would be to create a topic with a replication factor of 3, set min.insync.replicas to 2, and produce with request.required.acks of -1. This will ensure that the producer raises an exception if a majority of replicas do not receive a write.</td>
+      <td>When a producer sets acks to all, min.insync.replicas specifies the minimum number of replicas that must acknowledge a write for the write to be considered successful. If this minimum cannot be met, then the producer will raise an exception (either NotEnoughReplicas or NotEnoughReplicasAfterAppend).
```
Comment (gwenshap) [inline docs/configuration.html]: all should be in quotes, since it is an exact string value. "all".

**Comment by gwenshap** [comment]
thanks for the contribution. Just a minor correction.

**Comment by sasakitoa** [comment]
Thank you for review, @gwenshap.
I revised it.

**Comment by gwenshap** [comment]
LGTM. Thanks for contributing!

**Pull request closed without merging**